File: LJV-7000 Series - Prolifometer/LJV_communication_library1_4_2/LJV_communication_library1_4_2/Sample/CS/Source/Sample/Forms/GetStorageStatusForm.cs
//----------------------------------------------------------------------------- 
// <copyright file="GetStorageStatusForm.cs" company="KEYENCE">
//	 Copyright (c) 2013 KEYENCE CORPORATION.  All rights reserved.
// </copyright>
//----------------------------------------------------------------------------- 
using System;
using System.ComponentModel;
using System.Windows.Forms;

namespace LJV7_DllSampleAll.Forms
{
	public partial class GetStorageStatusForm : Form
	{
		#region Field
		/// <summary>
		/// Storage status request structure
		/// </summary>
		private LJV7IF_GET_STRAGE_STATUS_REQ _req;
		#endregion

		#region Property
		/// <summary>
		/// Storage status request structure
		/// </summary>
		public LJV7IF_GET_STRAGE_STATUS_REQ Req
		{
			get { return _req; }
		}
		#endregion

		#region Event
		/// <summary>
		/// Close start event
		/// </summary>
		/// <param name="e"></param>
		protected override void OnClosing(CancelEventArgs e)
		{
			if (DialogResult == DialogResult.OK)
			{
				try
				{
					_req.dwRdArea = Convert.ToUInt32(_txtboxInputValue.Text);
				}
				catch (Exception ex)
				{
					MessageBox.Show(this, ex.Message);
					e.Cancel = true;
					return;
				}
			}

			base.OnClosing(e);
		}
		#endregion

		#region Method
		/// <summary>
		/// Constructor
		/// </summary>
		public GetStorageStatusForm()
		{
			InitializeComponent();

			// Field initialization
			_req = new LJV7IF_GET_STRAGE_STATUS_REQ();
		}
		#endregion
	}
}

File: LJV-7000 Series - Prolifometer/LJV_communication_library1_4_2/LJV_communication_library1_4_2/Sample/VBNET/Source/Forms/GetStorageStatusForm.vb
'----------------------------------------------------------------------------- 
' <copyright file="GetStorageStatusForm.cs" company="KEYENCE">
'	 Copyright (c) 2013 KEYENCE CORPORATION.  All rights reserved.
' </copyright>
'----------------------------------------------------------------------------- 
Imports System.ComponentModel
Imports System.Windows.Forms

Namespace Forms
	Public Partial Class GetStorageStatusForm
		Inherits Form
		#Region "Field"
		''' <summary>
		''' Storage status request structure
		''' </summary>
		Private _req As LJV7IF_GET_STRAGE_STATUS_REQ
		#End Region

		#Region "Property"
		''' <summary>
		''' Storage status request structure
		''' </summary>
		Public ReadOnly Property Req() As LJV7IF_GET_STRAGE_STATUS_REQ
			Get
				Return _req
			End Get
		End Property
		#End Region

		#Region "Event"
		''' <summary>
		''' Close start event
		''' </summary>
		''' <param name="e"></param>
		Protected Overrides Sub OnClosing(e As CancelEventArgs)
			If DialogResult = DialogResult.OK Then
				Try
					_req.dwRdArea = Convert.ToUInt32(_txtboxInputValue.Text)
				Catch ex As Exception
					MessageBox.Show(Me, ex.Message)
					e.Cancel = True
					Return
				End Try
			End If

			MyBase.OnClosing(e)
		End Sub
		#End Region

		#Region "Method"
		''' <summary>
		''' Constructor
		''' </summary>
		Public Sub New()
			InitializeComponent()

			' Field initialization
			_req = New LJV7IF_GET_STRAGE_STATUS_REQ()
		End Sub
		#End Region
	End Class
End Namespace
